### PR TITLE
[ALLUXIO-2529] Update documentation of alluxio.worker.tieredstore.retry

### PR DESCRIPTION
**Pull request by xiaochang-wu**
https://alluxio.atlassian.net/browse/ALLUXIO-2529

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: docs/_data/table/en/worker-configuration.yml [hunk 1]
```diff
@@ -102,7 +102,7 @@ alluxio.worker.tieredstore.reserver.interval.ms:
   The time period (in milliseconds) of space reserver service, which keeps certain portion of
   available space on each layer.
 alluxio.worker.tieredstore.retry:
-  The number of retries that the worker to process blocks.
+  The number of retries that the worker processes blocks.
```
Comment (jsimsa) [inline docs/_data/table/en/worker-configuration.yml]: `The number of retries that the worker uses to process blocks.`

**Comment by jsimsa** [comment]
Jenkins, test this please

**Pull request merged**